src/main/target/KROOZX/target.h

```diff
@@ -95,7 +95,8 @@
 #define CURRENT_METER_OFFSET_DEFAULT   0
 
 #define USE_VCP
-//#define VBUS_SENSING_PIN        PA9
+#define USE_USB_DETECT
+//#define USB_DETECT_PIN          PA9
```

Comment:
This can probably be removed altogether. Since PA9 is used for UART1_TX, it is most certainly not connected to USB Vcc.